src/index.js

```diff
@@ -88,6 +88,10 @@ export default function copy(options = {}) {
         }
 
         for (const { src, dest } of copyTargets) {
+          try {
+            this.addWatchFile(src)
+          } catch { /* Cannot call addWatchFile after the build has finished. */ } 
```

Comment:
The `addWatchFile` method cannot be used after the build has finished, so is not available in all hooks. Rather than hard-coding a list of valid hooks that might change, I figured I'd just try/catch it. When it throws an error, it has the error message seen in the comment